.changeset/chilled-files-relate.md

```diff
@@ -0,0 +1,8 @@
+---
+'@rocket.chat/meteor': minor
+---
+
+Added two methods in the Apps Engine:
+
+1. A method to retrieve unread messages for a user from a specified room.
+2. A method to calculate the total number of unread messages for a user in a specified room.
```

Comment:
```suggestion
Added methods to the Apps-Engine that interact with unread messages
```

Please keep changeset messages short and concise. Detailed info should be put in dedicated documentation if necessary